node/sc/subbridge_test.go

```diff
@@ -40,6 +41,7 @@ func testNewSubBridge(t *testing.T) *SubBridge {
 	if err != nil {
 		t.Fatal(err)
 	}
+	defer os.RemoveAll(tempDir)
```

Comment:
I think the directory should be removed out of `testNewSubBridge` because the caller of the function uses the directory. Except for `TestSubBridge_handleMsg` all of test cases using the function remove the data calling `sBridge.removeData(t)`. @winnie-byun Could you check this again?

Reply:
Thanks for letting me know. I changed the place to call `RemoveAll`.
PTAL